build-tools/src/main/java/org/elasticsearch/gradle/testclusters/RunTask.java

```diff
@@ -139,12 +140,12 @@ public void beforeStart() {
         }
 
         for (ElasticsearchCluster cluster : getClusters()) {
-            cluster.getFirstNode().setHttpPort(String.valueOf(httpPort));
-            httpPort++;
-            cluster.getFirstNode().setTransportPort(String.valueOf(transportPort));
-            transportPort++;
             cluster.setPreserveDataDir(preserveData);
             for (ElasticsearchNode node : cluster.getNodes()) {
+                node.setHttpPort(String.valueOf(httpPort));
+                httpPort++;
+                node.setTransportPort(String.valueOf(transportPort));
+                transportPort++;
```

Comment:
this is here to ensure consistent port numbering (it worked before but picked a random port for the subsequent nodes)

Reply:
We can simplify this to just `node.setHttpPort(String.valueOf(httpPort++));`. Also, I wonder if it's worth detecting whether we are running multiple nodes, and then printing this information to the user. Otherwise it's quite hard to know what the ports for each of these cluster/nodes actually are.

Reply:
I can simplify the increment.  The information is printing in the server logs and is pretty guessable, 9200/9300 then 9201/9301 ...

Reply:
Fair enough. I just figured since it's getting spammed all together it might be hard to grok, we can add that later if folks find this cumbersome.

Reply:
incrementor simplified: [`e4649e5` (#89563)](https://github.com/elastic/elasticsearch/pull/89563/commits/e4649e50746bc28ceaecaba0f5cc71938f3e7157)